elasticsearch/tests/elasticsearch_test.py

```diff
@@ -82,6 +82,10 @@ def test_defaults():
             'name': 'node.ingest',
             'value': 'true'
         },
+        {
+            'name': 'sysctlInitContainer.enabled',
```

Comment:
I'm not going to run CI because I already know that this test will fail. This test is checking to see if there is an environment variable called `sysctlInitContainer.enabled` that is set to `true`. A test similar to this is what you are looking for: https://github.com/elastic/helm-charts/blob/288e5f72e76fd46254c43d7880b49957dc169b45/elasticsearch/tests/elasticsearch_test.py#L326-L335

Reply:
added 2 test :)